### DISCUSSION TITLE
Metadata Quarto issue or Pandoc Issue ? Or as-designed ?

### DISCUSSION THREAD

**Discussion by christophevx** [quarto-dev/quarto-cli#6497] [Q&A]
### Description

Metadata Quarto issue or Pandoc Issue ? Or as-designed ?

I am in a Quarto book project.

I would like to have the first part globalized for each chapter, but I don't want to write all everytime, so I made an include file (recap.qmd in my example).

I tested a lot of way to do that, but I am not able to include metadata values of the document in the render.

+ Root folder
  + 01 - First part
    + 01 - First chapter.qmd
    + 02 - Second chapter.qmd
  + 02 - Second Part
    + 01 - First chapter.qmd
    + 02 - Second chapter.qmd
  + _includes
    + recap.qmd
  + _quarto.yml

Here is my _quarto.yml

````qmd
book:
  author: An author
  chapters:
  - index.qmd
  - chapters:
    - "01 - First part/01 - First chapter.qmd"
    - "01 - First part/02 - Second chapter.qmd"
    part: "Introduction"
  - chapters:
    - "02 - Second Part/01 - First chapter.qmd"
    - "02 - Second Part/02 - Second chapter.qmd"
    part: "Another thing"
  date: 26/07/2023
  title: "What I want to do"
format:
  pdf:
    documentclass: book
    fontsize: 12pt
    header-includes:
    - \makeatletter
    - \@addtoreset{chapter}{part}
    - \makeatother
    mainfont: Imprima-Regular.ttf
    papersize: A4
    sansfont: Voces-Regular.ttf
project:
  output-dir: _book
  type: book
````

Here is my recap.qmd file

````qmd
Duration: {{< meta duration >}} minutes
````

Here is one of chapter file :

````qmd
---
title: "Title of the chapter"
duration: 4
type: V
---

In document : {{< meta duration >}} ==> Doesn't work

{{< include ../_includes/recap.qmd >}} ==> Doesn't work

````

The error I have when I render the project is : "WARNING: Unknown meta key duration specified in a metadata Shortcode"

I made the render with the simple command "quarto render --debug"

Can you tell me if if is a Quarto issue, a Pandoc issue or a "as-designed" feature ?

Or if I'm totally wrong with all of this ?

Thank you in advance.

**Comment by mcanouil** [comment]
Could you share a "working" project as a git repository?

Currently I am not fully sure to comprehend what you really want to do, or what "globalized" means.

Also, you seems to indicate that the meta shortcode does not work, but it does, meaning you have something particular in your setup.

<img width="1464" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/b306785e-4717-48ce-b056-82d2bee2f119">

PS: you really should not use `/` or spaces in filenames, _i.e._, `02 - Second Part/01 - First chapter.qmd` that's looking for trouble.

**Comment by christophevx** [comment]
Hello,

Thank you for answering.

You can have a repo here : https://github.com/christophevx/TestQuartoForm

You can try to do a "quarto render" in the root folder... I also attached the book in the _book folder, you can see the not working example on the 9th page and all the other ones...


PS : Yes, but it is automatically updated when I add a file in a folder ;)

**Comment by mcanouil** [comment]
I see now the issue. You think Quarto documents are separate documents for PDF books, but they are not as they are combined into one single tex file, with the main document being `index.qmd` which has no metadata thus the error.

I will open an issue.

**Reply by mcanouil** [reply]
Closing this in favour of #6532
